EIPS/eip-4910.md

```diff
@@ -4,7 +4,7 @@ title: Royalty Bearing NFTs
 description: Extension of EIP-721 to correctly define, process, and pay (hierarchical) onchain NFT royalties.
 author: Andreas Freund (@Therecanbeonlyone1969)
 discussions-to: https://ethereum-magicians.org/t/royalty-bearing-nfts/8453
-status: Review
+status: Last Call
```

Comment:
I'm not sure why the annotation didn't show up, but you'll need to add a `last-call-deadline` field, with a date about two weeks from when this PR gets merged.

Reply:
@SamWilsn ... just added it. You might want to update the EIP-1 description because it says that the editor adds that field. Hence, I did not add it.